### PR TITLE
Drop skip (needed for Windows Python 3.9+ packages)

### LINKED ISSUE #39: @conda-forge-admin, please re-render
@conda-forge-admin, please re-render.

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Drop the `skip` to build Windows packages for Python 3.9+. Also added an upstream patch to resolve Python 3.10 Windows build issues. This is needed for the Python 3.10 migration where Windows packages are missed. Plus for packages that need this on Python 3.9.

<hr>

Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #39.

Here's a checklist to do before merging.
- [x] Bump the build number if needed.

Fixes #39

Follow-up to PR ( https://github.com/conda-forge/jpype1-feedstock/pull/38 )

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,6 +8,13 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 4fc27dba89750cb0c9d692466341ce60c0fe86a16051091cb5347a37cf884151
+  patches:
+    ###########################################################
+    # Include patch for Windows Python 3.10 compiler error.   #
+    #                                                         #
+    # xref: https://github.com/jpype-project/jpype/pull/1011  #
+    ###########################################################
+    - patches/PR_1011.patch
```
Comment (jakirkham) [inline recipe/meta.yaml:17]: xref: https://github.com/jpype-project/jpype/pull/1011

Comment (marscher) [reply recipe/meta.yaml:17]: thank you so much for taking care of this!

Comment (jakirkham) [reply recipe/meta.yaml:17]: Happy to help 🙂

**Pull request merged**